spec/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -423,20 +430,21 @@ Note that the full packet is not stored in the state of the chain - merely a sho
 
 ```typescript
 function sendPacket(packet: Packet) {
-  channel = get(channelKey(packet.sourceConnection, packet.sourceChannel))
+  channel = get(channelKey(packet.sourcePort, packet.sourceChannel))
   assert(channel.state === OPEN)
-  assert(authenticate(get(portKey(channel.portIdentifier))))
+  assert(authenticate(get(portKey(packet.sourcePort))))
+  assert(packet.destPort === channel.counterpartyPortIdentifier)
   assert(packet.destChannel === channel.counterpartyChannelIdentifier)
-  connection = get(connectionKey(packet.sourceConnection))
+  connection = get(connectionKey(packet.connectionHops[0]))
   assert(connection.state === OPEN)
-  assert(packet.destConnection === connection.counterpartyConnectionIdentifier)
+  assert(packet.connectionHops === channel.connectionHops)
```

Comment:
Will this be true with number of connection hops > 2? Is the plan to have all hops in the packet?

Reply:
Not necessary; it depends on the details of how we store the routing table.

For now it is true though.